tests/rules/test_drmemory_no_app_specified.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from thefuck.rules.drmemory_no_dashdash import match, get_new_command
+from thefuck.types import Command
+
+
+@pytest.mark.parametrize('command', [
+    Command('drmemory a.out', 'Usage: drmemory [options] --'),
+    Command('/etc/bin/drmemory test.exe', 'Usage: drmemory [options] --'),
+])
+def test_match(command):
+    assert match(command)
+
+
+@pytest.mark.parametrize('command', [
+    Command('drmemory -- a.out', ''),
+    Command('/etc/bin/drmemory -- test.exe', ''),
+])
+def test_not_match(command):
+    assert not match(command)
+
+
+@pytest.mark.parametrize('command, new_command', [
+    (Command('drmemory a.out', ''), 'drmemory -- a.out'),
+    (Command('/etc/bin/drmemory test.exe', ''), '/etc/bin/drmemory -- test.exe'),
```

Comment:
Dr. Memory's docs cites the following usage:

```
drmemory.exe -logdir c:/logs -- c:/path/to/my/app
```

The rule does not correctly fix the following wrong usage:

```
drmemory.exe -logdir c:/logs c:/path/to/my/app
```